vms/shift/templates/shift/edit_hours.html

```diff
@@ -1,6 +1,22 @@
 {% extends "vms/base.html" %}
 
 {% block content %}
+<script type="text/javascript">
```

Comment:
I think, we have no such file, I have to create a new one. I will do it, but how can I import this file? 
```
<script type="text/javascript"
        src="http://example.com/scriptFile.js"></script>
```
Does it work in Django? (This was my confusing question :wink:)